components/bounty/funding-section/fund-modal.tsx

```diff
@@ -0,0 +1,163 @@
+import { useContext, useEffect, useState } from "react";
+
+import { useTranslation } from "next-i18next";
+
+import LockedIcon from "assets/icons/locked-icon";
+
+import FundingProgress from "components/bounty/funding-section/funding-progress";
+import Button from "components/button";
+import InputWithBalance from "components/input-with-balance";
+import Modal from "components/modal";
+
+import { ApplicationContext } from "contexts/application";
+import { useIssue } from "contexts/issue";
+import { toastError, toastSuccess } from "contexts/reducers/add-toast";
+
+import { formatNumberToCurrency } from "helpers/formatNumber";
+
+import useBepro from "x-hooks/use-bepro";
+import useERC20 from "x-hooks/use-erc20";
+
+import { Amount, CaptionMedium, RowWithTwoColumns } from "./minimals";
+
+export default function FundModal({
+  show = false,
+  onCloseClick
+}) {
+  const { t } = useTranslation(["common", "funding", "bounty"]);
+
+  const [amountToFund, setAmountToFund] = useState(0);
+  const [rewardPreview, setRewardPreview] = useState(0);
+  const [isExecuting, setIsExecuting] = useState(false);
+
+  const { handleFundBounty } = useBepro();
+  const { dispatch } = useContext(ApplicationContext);
+  const { allowance, balance, setAddress, approve } = useERC20();
+  const { activeIssue, networkIssue, getNetworkIssue } = useIssue();
+
+  const bountyId = activeIssue?.contractId || networkIssue?.id || "XX";
+  const fundBtnDisabled = 
+    isExecuting || !amountToFund || amountToFund + networkIssue?.fundedAmount > networkIssue?.fundingAmount;
```

Comment:
shouldn't this be `>=` instead ?

Reply:
It verify if the value entered by the user is greater than the available amount to fund.
If a funding request is asking for 100 BEPRO and already received 50 BEPRO, new funders can't fund more than 50 BEPRO.